### PR TITLE
SDKS-1403 Sample App for Documentation

### PR DESCRIPTION
**Pull request by witrisna**

### CODE REVIEW

**Review by spetrov — DISMISSED** [review]
I tried the sample app and it looks good to me! 👍🏻

File: samples/quickstart/src/main/java/org/forgerock/quickstart/NodeDialogFragment.java [hunk 1]
```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2020 ForgeRock. All rights reserved.
```
Comment (rodrigoareis) [inline samples/quickstart/src/main/java/org/forgerock/quickstart/NodeDialogFragment.java:2]: 2021?

File: samples/quickstart/src/main/res/layout/fragment_node.xml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2020 ForgeRock. All rights reserved.
```
Comment (rodrigoareis) [inline samples/quickstart/src/main/res/layout/fragment_node.xml:3]: 2021?

**Review by rodrigoareis — CHANGES_REQUESTED** [review]
Looks good to me. Minor copyright issues.

File: samples/quickstart/src/main/java/org/forgerock/quickstart/NodeDialogFragment.java (hunk 1 above)
Comment (witrisna) [reply samples/quickstart/src/main/java/org/forgerock/quickstart/NodeDialogFragment.java]: Updated.

File: samples/quickstart/src/main/res/layout/fragment_node.xml (hunk 2 above)
Comment (witrisna) [reply samples/quickstart/src/main/res/layout/fragment_node.xml]: Updated

**Review by spetrov — APPROVED** [review]

**Review by spetrov — APPROVED** [review]

**Pull request merged**